opensearch_py_ml/ml_commons/ml_commons_client.py

```diff
@@ -264,7 +264,7 @@ def _send_model_info(self, model_meta_json: dict):
         )
         end = time.time() + TIMEOUT  # timeout seconds
         task_flag = False
-        while not task_flag or time.time() < end:
+        while not task_flag and time.time() < end:
```

Comment:
I believe we should consider the current condition. If the task status is anything other than 'created,' there's no need for us to continuously check the task status. If this condition is met before the set time limit, we can proceed further; otherwise, we will wait until the specified time has passed.